### DISCUSSION TITLE
Identification of transcription factor targets

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#819] [Feature Requests]
Hi,

 Is there any way of generating regulatory network showing predicted targets of the Transcription factors correlated with the enriched motifs (GIM_MM)?

Thanks!

**Comment by rcorces** [comment]
We do not currently have this as a feature in ArchR. I've changed your label to Feature Request and we will keep this in mind in the future.

**Comment by sylestiel** [comment]
Thank you!
